test/check-machines-nics

```diff
@@ -139,6 +140,20 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_not_present(".pf-c-modal-box")
         b.wait_not_present("#vm-subVmTest1-network-1-mac")
 
+        self.performAction("subVmTest1", "off")
```

Comment:
This is also tested at https://github.com/cockpit-project/cockpit-machines/blob/main/test/check-machines-nics#L266 

If you want to have it better organized we can move that deletion code into the testNICDelete test.

Reply:
Yeah, I think that is also better for align this verify case with polarion cases. So should I need to move the case in this PR? And should I need some other modifications?

Reply:
Moving the relevant code is enough.

Reply:
It seems that the new added check is the same with the original one, so I just remove the original check

Reply:
Actually the check is not the same. The previous code was covering  https://github.com/cockpit-project/cockpit/commit/7c041b8f8574cfadaf9ea4272d95da4f1f7cca80, so https://bugzilla.redhat.com/show_bug.cgi?id=1791543

There need to be two interfaces for this.

Reply:
Thanks. This issues is too old, so I provision a rhel 8.2 machine to reproduce this issue. And for reproducing the second "Actual results" of the issue, we need add another two network interfaces, and the type of the second one must be bridge. So, according above, I add a few lines. Please check and remind me if something is wrong, many thanks